src/jailer/src/main.rs

```diff
@@ -329,7 +324,11 @@ fn main() {
             if let Some(help) = arg_parser.arguments().value_as_bool("help") {
                 if help {
                     println!("Jailer v{}\n", JAILER_VERSION);
-                    println!("{}", arg_parser.formatted_help());
+                    println!("{}\n", arg_parser.formatted_help());
+                    println!(
```

Comment:
Since this is related to help message, should we add it to the `formatted_help` method? Extra args are not relevant in the context of `firecracker` alone, so I would specialize the `ArgParser` as well, as in `ArgParser` which is aware of extra args or not.

Reply:
Nevermind, to late. 😅